src/contracts/interfaces/IEigenPod.sol

```diff
@@ -123,16 +125,13 @@ interface IEigenPod {
      *        Must be within `VERIFY_OVERCOMMITTED_WINDOW_BLOCKS` of the current block.
      * @param validatorIndex is the index of the validator being proven, refer to consensus specs 
      * @param proofs is the proof of the validator's balance and validatorFields in the balance tree and the balanceRoot to prove for
-     * @param beaconChainETHStrategyIndex is the index of the beaconChainETHStrategy for the pod owner for the callback to 
```

Comment:
in `verifyWithdrawalCredentials`, we do not set 2 fields of the validatorInfo struct?

Reply:
I see 'validatorIndex', 'restakedBalanceGwei', and 'status' getting updated, but _not_ 'mostRecentBalanceUpdateSlot' -- this is just one field, but kind of awkward / doesn't seem desired.
@Sidu28 can you confirm if bug?